commands/issue.go

```diff
@@ -603,6 +618,58 @@ text is the title and the rest is the description.`, project))
 	messageBuilder.Cleanup()
 }
 
+func addComment(cmd *Command, args *Args) {
+	issueNumber := ""
+	if args.ParamsSize() > 0 {
+		issueNumber = args.GetParam(0)
+	}
+	if issueNumber == "" {
+		utils.Check(cmd.UsageError(""))
+	}
+
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.MainProject()
+	utils.Check(err)
+
+	gh := github.NewClient(project.Host)
+
+	commentBuilder := &github.CommentBuilder{
+		Filename: "ISSUE_EDITMSG",
```

Comment:
So it doesn't conflict with the `ISSUE_EDITMSG` file potentially left over from creating an issue:
```suggestion
		Filename: "ISSUE_COMMENT_EDITMSG",
```